website/src/components/ElevatorPitch/index.tsx

```diff
@@ -70,15 +70,15 @@ export default function ElevatorPitch() {
   };
 
   return (
-    <div className="flex w-full lg:flex-row flex-col justify-center gap-6 lg:gap-16 xl:gap-36 items-center">
+    <div className="flex w-full h-fit lg:flex-row flex-col justify-center gap-6 lg:gap-12 xl:gap-20 items-center max-w-screen-xl ">
       <div className="flex flex-col w-full h-full justify-between lg:w-[480px] xl:w-[580px]">
         <div>
           <h6 className="uppercase text-sm font-semibold text-primary-450 tracking-wide mb-2">
             Stay Connected
           </h6>
           <div className="mb-2 text-3xl md:text-4xl lg:text-5xl ">
             <h3
-              className={`leading-tight text-pretty text-left tracking-tight font-bold inline-block ${manrope.className}`}
+              className={`leading-none text-pretty text-left tracking-tight font-bold inline-block ${manrope.className}`}
```

Comment:
I added `leading-tight` so the text wouldn't touch here. I think that looks better?

Reply:
hmm maybe we can get fb from the team?